cli/command/load_balancer.go

```diff
@@ -10,6 +18,497 @@ func NewLoadBalancerCommand(b *CommandBase) *LoadBalancerCommand {
+	if err != nil {
+		return fmt.Errorf("'%s' is not a valid integer", args["HOST_PORT"])
+	}
+
+	loadBalancerID, err := l.resolveSingleEntityIDHelper("load_balancer", args["LOADBALANCER_NAME"])
+	if err != nil {
+		return err
+	}
+
+	loadBalancer, err := l.client.ReadLoadBalancer(loadBalancerID)
+	if err != nil {
+		return err
+	}
+
+	var exists bool
+	for i, p := range loadBalancer.Ports {
+		if p.HostPort == port {
+			loadBalancer.Ports = append(loadBalancer.Ports[:i], loadBalancer.Ports[i+1:]...)
+			exists = true
+		}
+	}
+
+	if !exists {
+		return fmt.Errorf("Host port '%v' doesn't exist on this Load Balancer", port)
+	}
+
+	req := models.UpdateLoadBalancerRequest{
+		LoadBalancerID: loadBalancerID,
+		Ports:          &loadBalancer.Ports,
+		HealthCheck:    nil,
+	}
+
+	jobID, err := l.client.UpdateLoadBalancer(req)
+	if err != nil {
+		return err
+	}
+
+	onCompleteFn := func(loadBalancerID string) error {
+		loadBalancer, err := l.client.ReadLoadBalancer(loadBalancerID)
+		if err != nil {
+			return err
+		}
+
+		l.printer.StopSpinner()
+		return l.printer.PrintLoadBalancers(loadBalancer)
+	}
+
+	return l.waitOnJobHelper(c, jobID, "Dropping port from load balancer", onCompleteFn)
+}
+
+func (l *LoadBalancerCommand) healthcheck(c *cli.Context) error {
+	args, err := extractArgs(c.Args(), "LOADBALANCER_NAME")
+	if err != nil {
+		return err
+	}
+
+	requiresUpdate := false
+	healthCheck := models.HealthCheck{}
+
+	if target := c.String("healthcheck-target"); target != "" {
+		target, err := parseTarget(target)
+		if err != nil {
+			return err
+		}
+
+		requiresUpdate = true
+		healthCheck.Target = target
+	}
+
+	if interval := c.String("healthcheck-interval"); interval != "" {
+		interval, err := strconv.Atoi(interval)
+		if err != nil {
+			return err
+		}
+
+		requiresUpdate = true
+		healthCheck.Interval = interval
+	}
+
+	if timeout := c.String("healthcheck-timeout"); timeout != "" {
+		timeout, err := strconv.Atoi(timeout)
+		if err != nil {
+			return err
+		}
+
+		requiresUpdate = true
+		healthCheck.Timeout = timeout
+	}
+
+	if healthyThreshold := c.String("healthy-threshold"); healthyThreshold != "" {
+		healthyThreshold, err := strconv.Atoi(healthyThreshold)
+		if err != nil {
+			return err
+		}
+
+		requiresUpdate = true
+		healthCheck.HealthyThreshold = healthyThreshold
+	}
+
+	if unhealthyThreshold := c.String("unhealthy-threshold"); unhealthyThreshold != "" {
+		unhealthyThreshold, err := strconv.Atoi(unhealthyThreshold)
+		if err != nil {
+			return err
+		}
+
+		requiresUpdate = true
+		healthCheck.UnhealthyThreshold = unhealthyThreshold
+	}
+
+	loadBalancerID, err := l.resolveSingleEntityIDHelper("load_balancer", args["LOADBALANCER_NAME"])
+	if err != nil {
+		return err
+	}
+
+	loadBalancer, err := l.client.ReadLoadBalancer(loadBalancerID)
+	if err != nil {
+		return err
+	}
+
+	if requiresUpdate {
+		if healthCheck.Target != "" {
+			loadBalancer.HealthCheck.Target = healthCheck.Target
```

Comment:
[nit] ditto as above - not assigning a new variable for the update healthcheck is a little odd. Seems clearer to assign/update a new one, e.g.
```go
loadBalancer, err := client.ReadLoadBalancer(...)
...
healthCheck := loadBalancer.HealthCheck
...
if timeout := c.Int("healthcheck-timeout"); timeout != 0 {
    healthCheck.Timeout = timeout
    requiredUpdate = true
}
...
req := models.UpdateLoadBalancerRequest{
    LoadBalancerID: loadBalancerID,
    HealthCheck: &healthCheck
}

...
```

Reply:
:white_check_mark: